addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -58,7 +58,7 @@
             <field name="model">mailing.mailing</field>
             <field name="arch" type="xml">
                 <form string="Mailing" class="o_mass_mailing_mailing_form">
-                    <header class="pt-3">
```

Comment:
This class was supposed to solve another issue apparently... could you check? hint: https://github.com/odoo/odoo/commit/fabf9e97037d71f767e09fcc4f5aeffb2b71f05a

Reply:
I had a look at the commit but I'm not quite sure I understood why we added that `pt-3`.

I believe we added it to separate the statusbar from the `border-bottom` of the CP and replicate the padding we use on the `form_sheet`, but this header is also used in `mass_mailing_sms` which creates an excessive `padding-top`

<img width="1728" alt="image" src="https://github.com/odoo/odoo/assets/128030743/c8f1ad8e-7fd0-4f8d-b877-fa67599e89f9">